using_images/sti_images/python.adoc

```diff
@@ -12,8 +12,7 @@ toc::[]
 == Overview
 The OpenShift https://github.com/openshift/sti-python/tree/master/3.3[Python]
 image repository contains the sources and Dockerfiles for building various
-versions of Python platform images. The resulting images can be run either by
-Docker or using https://github.com/openshift/source-to-image[STI].
+versions of Python platform images. These images can be run either by Docker or with the link:../../architecture/core_objects/builds.html#sti-build[STI build strategy].
```

Comment:
Looking at this, I see we have a couple of problems, I'll tell you the idea and you can put that into beautiful words :smile: The general idea with those image (I'm talking here about all sti image) is they provide the needed tools to build the final image that runs your application. In short term the flow is following:
application source + sti builder image (eg. python) = docker image
where:
- application source is what developer provides
- sti builder image is what you're writing about
- docker image is the resulting image containing your application you can either run locally using docker or in OpenShift
STI itself can be seen as the plus sign between app source and builder image, it's responsible for combining the two. 
Based on what I said, the above description should more or less mark the difference between the builder image and the resulting docker image. The first you use with STI, the latter you run with docker or inside of OpenShift.